lib/db.js

```diff
@@ -787,7 +787,7 @@ module.exports = function (
       params: params
     })
 
-    return this.pool.get('/securityEvents', params)
```

Comment:
`pool.get` only supports a single parameter, so I converted this URL to be more RESTful.